src/app/games/tablut/TablutFailure.ts

```diff
@@ -1,12 +1,12 @@
 export class TablutFailure {
 
-    public static readonly CHOOSE_OWN_PIECE: string = $localize`Pour votre premier clic, choisissez une de vos pièces.`;
+    public static readonly CHOOSE_OWN_PIECE: string = $localize`For your first click, pick one of your pieces.`;
 
-    public static readonly LANDING_ON_OCCUPIED_CASE: string = $localize`landing on occupied case.`;
+    public static readonly LANDING_ON_OCCUPIED_CASE: string = $localize`You cannot land on an occupied case.`;
 
-    public static readonly THRONE_IS_LEFT_FOR_GOOD: string = $localize`Throne is left for good.`
+    public static readonly THRONE_IS_LEFT_FOR_GOOD: string = $localize`Once you left the central throne, you cannot return to it.`;
```

Comment:
once you leave ?

Reply:
discutable : une fois que vous avez quitté le trone, ou une fois que vous quittez le trone.
Pour moi le "left" a plus de sens vu que quand ce message s'affiche, le trone a déjà été left, dans le passé

Reply:
Oui allez, va pour "une fois que vous avez quitté"
Ceci dit, on a pas encore de jeu où cette règle est à true :p